src/reducers/app.js

```diff
@@ -170,6 +170,8 @@ const lastVisibleThreadTabSlug: Reducer<TabSlug> = (
         return action.selectedTab;
       }
       return state;
+    case 'FOCUS_CALL_TREE':
+      return 'calltree';
```

Comment:
I'm not a huge fan of this `lastVisibleThreadTabSlug`, it's missing some corner cases (for example when navigating using the URL only, either loading a URL with another slug than `calltree` at load time, or navigating with back/forward). But I didn't want to change all of this now :-)

(reminder: this state links the network chart and the network track, so that when changing the track back to another track, we go back at the previous selected tab too)